meta-refkit/conf/distro/include/refkit-ci.inc

```diff
@@ -40,6 +40,12 @@ TEST_TARGET = "simpleremote"
 # SDK defaults:
 SDKMACHINE ?= "x86_64"
 
+# HOSTTOOLS setup:
+HOSTTOOLS_remove = "hostname ip vi"
```

Comment:
Why we need to *remove* things? If they are not truly needed, then OE-core shouldn't have them in HOSTTOOLS either.